packages/common/container-definitions/src/audience.ts

```diff
@@ -36,12 +36,13 @@ export interface IAudienceOwner extends IAudience {
 export interface IAudienceEvents extends IEvent {
 	/**
 	 * "addMember" event is raised when a new user joins collaborative session.
```

Comment:
Nit (since they're individual APIs now):
```suggestion
	 * Raised when a new user joins collaborative session.
```

Reply:
I would recommend adding docs for the callback parameters as well, while we're here.

Reply:
And I would recommend tagging these with the `@eventProperty` TSDoc tag too :)